daemon/remote/cmd/cmuxd-remote/main.go

```
@@ -1605,22 +1662,49 @@ func runRPCServerWithReader(
 }
 
 func dialPersistentDaemon(socketPath string, token string) (net.Conn, error) {
+	return dialPersistentDaemonWithBridgeLease(socketPath, token, "")
+}
+
+func dialPersistentDaemonWithBridgeLease(socketPath string, token string, bridgeLeaseID string) (net.Conn, error) {
 	conn, err := net.DialTimeout("unix", socketPath, 2*time.Second)
```

golangci-lint (2.12.2)

[error] net.DialTimeout must not be called. use (*net.Dialer).DialContext with (*net.Dialer).Timeout

(noctx)